<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon/res.qrc">
    <normaloff>:/find/search.png</normaloff>:/find/search.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="cb_searchType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Step</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btn_find">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icon/res.qrc">
         <normaloff>:/find/search.png</normaloff>:/find/search.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edt_searchText">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edt_searchText</tabstop>
  <tabstop>btn_find</tabstop>
  <tabstop>cb_searchType</tabstop>
 </tabstops>
 <resources>
  <include location="icon/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_searchText</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_find</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
